cfgov/prepaid_agreements/jinja2/prepaid_agreements/detail.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/prepaid-agreements/css/main.css') }}">
+{%- endblock css %}
+
+{% block title -%}
+    {{ product.name }} | Consumer Financial Protection Bureau
+{%- endblock title %}
+
+{% block hero %}
+<div class="wrapper content_wrapper">
+
+    {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
+    {% set breadcrumb_items = [{
+        'href': '/prepaid-agreements/',
+        'title': 'Prepaid Product Agreements Database'
```

Comment:
The title of the new index page shows up as "Prepaid Agreements Search" in the browser title bar (from the `<title>` tag), "Prepaid account agreements database" on the page itself, and "Prepaid Product Agreements Database" here in the link from the detail page. Could these be made consistent?

Reply:
@sonnakim any thoughts here?

Reply:
Yes - the outcome of this is sort of awaiting clearance, but let's unify for now: 

- Title tag: Prepaid Product Agreements Database
- H1: Prepaid product agreements database
- Detail page breadcrumb: Prepaid Product Agreements Database

@richaagarwal @virginiacc

Reply:
This should now be standardized as @sonnakim describes.